instrumentation/opentelemetry-sdk-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetrysdk/OpenTelemetrySdkInstrumentation.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.opentelemetrysdk;
+
+import static net.bytebuddy.matcher.ElementMatchers.isMethod;
+import static net.bytebuddy.matcher.ElementMatchers.isPublic;
+import static net.bytebuddy.matcher.ElementMatchers.isStatic;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
+
+import io.opentelemetry.javaagent.tooling.TypeInstrumentation;
+import io.opentelemetry.sdk.trace.TracerSdkManagement;
+import java.util.Collections;
+import java.util.Map;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+public class OpenTelemetrySdkInstrumentation implements TypeInstrumentation {
+  @Override
+  public ElementMatcher<? super TypeDescription> typeMatcher() {
+    return named("io.opentelemetry.sdk.OpenTelemetrySdk");
+  }
+
+  @Override
+  public Map<? extends ElementMatcher<? super MethodDescription>, String> transformers() {
+    return Collections.singletonMap(
+        isMethod()
+            .and(isPublic())
+            .and(isStatic())
+            .and(named("getGlobalTracerManagement"))
+            .and(takesArguments(0)),
+        OpenTelemetrySdkInstrumentation.class.getName() + "$GetGlobalTracerManagementAdvice");
+  }
+
+  public static class GetGlobalTracerManagementAdvice {
+
+    @Advice.OnMethodEnter(skipOn = Advice.OnDefaultValue.class)
+    public static Object onEnter() {
+      return null;
+    }
+
+    @Advice.OnMethodExit(onThrowable = Throwable.class, suppress = Throwable.class)
+    public static void methodExit(@Advice.Return(readOnly = false) TracerSdkManagement management) {
+      NoopTracerManagement.logCannotUseTracerManagementWarning();
+      management = NoopTracerManagement.INSTANCE;
```

Comment:
do we need to return no-op?

Reply:
What do we return if not a no-op? It seems to also fit well with the "not supported" aspect, since we don't support it we make it a no-op